functions/check_deploy.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
A docs file listing all the HTTP endpoints, the URL patterns that they accept and the expected variety of return codes would be helpful for all these functions